free-1.md

```diff
@@ -10,41 +10,120 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My first idea is for a DSL that allows the user to create simple looping 
+animations. This DSL meets the need of inspiring creativity through code! It
+allows users to create animated art work through simple programming. Animation 
+can be a very tedious and intimidating process for those who don't have strong 
+drawing skills or experience with the Adobe Creative Suite. In other words, 
+animation seems out of reach for those who don't really have an art background 
+and are just casually interested in animation. This DSL allows for users to 
+create cool animations with very little to no background in drawing or coding! 
+I imagine it would be a great learning tool for kids, inspiring interest in 
+the fields of both Computer Science and Art.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would allow the user to more easily create looping animations. For 
+example, it's likely easier to define and add animations to large number of 
+shapes through code than through manually drawing them or adding new shapes
+through some sort of GUI. It addresses the desire to create interesting 
+looping patterns and movements with a relatively little investment in time.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I'm excited by this idea because I'm personally interested in learning 
+about animation but have very little experience with it! I created very 
+minor animations using Unity this summer, but found it to be a rather 
```

Comment:
I see the similarity as well - would the language try to be similar to ContextFree in that it does recursion/symmetry well, or arbitrary shapes well? Or would it try to do both?